pkgs/applications/graphics/meh/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, libX11, libXext, libjpeg, libpng, giflib }:
+
+stdenv.mkDerivation rec {
+  name = "meh-0.3";
+
+  src = fetchFromGitHub {
+	 owner = "jhawthorn";
+	 repo = "meh";
+     rev = "4ab1c75f97cb70543db388b3ed99bcfb7e94c758";
```

Comment:
You can probably do `rev = "v0.3"` here. For convenience, consider factoring out a dedicated version attribute, to be reused in `name` and `rev`.